src/components/timecard/edit-shift-hours/EditShiftHours.jsx

```diff
@@ -141,6 +141,7 @@ const EditShiftHours = ({
     dayjs.extend(utc);
 
     setSummaryMessages([]);
+    setSummaryErrors({});
```

Comment:
Now that the UI review is merged this should be an array here []. Update from main and see if everything still plays ball with this change in.